state/remote/s3.go

```diff
@@ -26,19 +26,30 @@ func s3Factory(conf map[string]string) (Client, error) {
 	if !ok {
 		regionName = os.Getenv("AWS_DEFAULT_REGION")
 		if regionName == "" {
-			return nil, fmt.Errorf("missing 'region' configuration or AWS_DEFAULT_REGION environment variable")
+			return nil, fmt.Errorf(
+				"missing 'region' configuration or AWS_DEFAULT_REGION environment variable")
 		}
 	}
 
 	accessKeyId := conf["access_key"]
 	secretAccessKey := conf["secret_key"]
 
-	credentialsProvider := credentials.NewStaticCredentials(accessKeyId, secretAccessKey, "")
+	credentialsProvider := credentials.NewChainCredentials([]credentials.Provider{
+		&credentials.StaticProvider{Value: credentials.Value{
+			AccessKeyID:     accessKeyId,
+			SecretAccessKey: secretAccessKey,
+			SessionToken:    "",
```

Comment:
There is actually a token config option, I see no reason to set this to empty string.